src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/DeadlineSteps.java

```diff
@@ -107,7 +108,7 @@ public void questionsDeadlinesHaveBeenSuccessfullyExtended(String questionExpect
         List<QuestionResponse> questionResponses = allQuestionsResponse.getQuestions();
 
         Calendar expectedExpiryDate = new GregorianCalendar(TimeZone.getTimeZone("GMT"));
-        expectedExpiryDate.add(Calendar.DAY_OF_YEAR, 7 + 7); // deadline for answer + extension
+        expectedExpiryDate.add(Calendar.DAY_OF_YEAR, ExpiryCalendar.deadlineExtensionDays + ExpiryCalendar.deadlineExtensionDays); // deadline for answer + extension
```

Comment:
ExpiryCalendar.deadlineExtensionDays * 2 would be nicer

Reply:
Done